lighthouse-cli/bin.ts

```diff
@@ -249,6 +249,36 @@ function handleError(err: LighthouseError) {
   }
 }
 
+function saveResults(results: Results,
+                     flags: {output: any, outputPath: string, saveArtifacts: boolean, saveAssets: boolean}) {
+    let promise = Promise.resolve(results);
+    const cwd = process.cwd();
+    const configuredPath = !flags.outputPath || flags.outputPath === 'stdout' ?
+        assetSaver.getFilenamePrefix(results) :
+        flags.outputPath.replace(/\.\w{2,4}$/, '');
+    const resolvedPath = path.resolve(cwd, configuredPath);
+    const pathWithBasename = resolvedPath.includes(cwd) ?
+        resolvedPath.slice(cwd.length + 1) : resolvedPath;
```

Comment:
my read is that resolvedPath always includes cwd because we just did path.resolve on it.
and so we'll always slice,
taking the absolute file path back to a relative one.

This is 1) the same as `path.relative(cwd, resolvedPath)` AFAICT. but 2) i don't know why we nuke it, since its a more absolute location.